### DISCUSSION TITLE
How does the sectors generate?

### DISCUSSION THREAD

**Discussion by TheUns33n** [vonbeardy/hd2galaxy-issues#58] [Q&A] [answered]
How does the game/your map generate the sectors like it does?

**Comment by vonbeardy** [comment, marked as answer]
Not sure exactly what you are after here but the sector information is pretty vague from the official API other than they provide an identification number for each sector (not the names) which is then associated to each planet. The current visual sectors you are seeing on the site have been generously produced by @josh-allison who created them in a SVG format so I have the ability to dynamically change the styling with code.
